Only in xmlrpc-epi-0.50.fixed: Makefile
diff -ru --exclude missing --exclude ltmain.sh --exclude install-sh --exclude config.sub --exclude config.guess --exclude INSTALL --exclude COPYING --exclude mkinstalldirs --exclude Makfile --exclude Makefile.in --exclude aclocal.m4 --exclude configure --exclude '*.o' --exclude '*~' --exclude '*.lo' --exclude .libs --exclude .deps xmlrpc-epi-0.50/Makefile.am xmlrpc-epi-0.50.fixed/Makefile.am
--- xmlrpc-epi-0.50/Makefile.am	Tue Apr 10 03:37:40 2001
+++ xmlrpc-epi-0.50.fixed/Makefile.am	Thu Apr 11 16:26:38 2002
@@ -1,4 +1,4 @@
-SUBDIRS = expat src sample
+SUBDIRS = src sample
 
 CHANGE_LOG_GEN_BIN = scripts/cvs2cl.pl
 CHANGE_LOG_GEN_FLAGS = -t -r -b --accum -I ChangeLog --gmt --prune
Only in xmlrpc-epi-0.50.fixed: autom4te.cache
Only in xmlrpc-epi-0.50.fixed: config.log
Only in xmlrpc-epi-0.50.fixed: config.status
diff -ru --exclude missing --exclude ltmain.sh --exclude install-sh --exclude config.sub --exclude config.guess --exclude INSTALL --exclude COPYING --exclude mkinstalldirs --exclude Makfile --exclude Makefile.in --exclude aclocal.m4 --exclude configure --exclude '*.o' --exclude '*~' --exclude '*.lo' --exclude .libs --exclude .deps xmlrpc-epi-0.50/configure.in xmlrpc-epi-0.50.fixed/configure.in
--- xmlrpc-epi-0.50/configure.in	Thu Sep 27 22:56:32 2001
+++ xmlrpc-epi-0.50.fixed/configure.in	Thu Apr 11 16:51:11 2002
@@ -13,13 +13,10 @@
 AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
-AC_PROG_RANLIB
-
-LIBS="-L/usr/local/lib"
 
 dnl Checks for header files.
 AC_HEADER_STDC
-AC_CHECK_HEADERS(fcntl.h malloc.h unistd.h)
+AC_CHECK_HEADERS(fcntl.h malloc.h unistd.h xmlparse.h expat.h)
 
 
 dnl Checks for typedefs, structures, and compiler characteristics.
@@ -35,8 +32,9 @@
 AC_FUNC_STRFTIME
 
 AC_CHECK_FUNCS(mktime strstr)
+AC_CHECK_LIB(expat, XML_Parse)
 
-AC_OUTPUT(src/Makefile Makefile sample/Makefile expat/Makefile expat/xmltok/Makefile expat/xmlparse/Makefile)
+AC_OUTPUT(src/Makefile Makefile sample/Makefile)
 
 
 
Only in xmlrpc-epi-0.50.fixed: libtool
Only in xmlrpc-epi-0.50.fixed/sample: Makefile
diff -ru --exclude missing --exclude ltmain.sh --exclude install-sh --exclude config.sub --exclude config.guess --exclude INSTALL --exclude COPYING --exclude mkinstalldirs --exclude Makfile --exclude Makefile.in --exclude aclocal.m4 --exclude configure --exclude '*.o' --exclude '*~' --exclude '*.lo' --exclude .libs --exclude .deps xmlrpc-epi-0.50/sample/Makefile.am xmlrpc-epi-0.50.fixed/sample/Makefile.am
--- xmlrpc-epi-0.50/sample/Makefile.am	Fri Aug 31 23:38:27 2001
+++ xmlrpc-epi-0.50.fixed/sample/Makefile.am	Thu Apr 11 16:58:25 2002
@@ -1,25 +1,25 @@
-INCLUDES=-I../src -I../expat/xmltok -I../expat/xmlparse
+INCLUDES=-I../src
 
 bin_PROGRAMS = sample client server server_compliance_test memtest hello_client hello_server
 
 sample_SOURCES = sample.c
-sample_LDADD = -L@prefix@/lib ../src/libxmlrpc.la
+sample_LDADD = ../src/libxmlrpc.la
 
 client_SOURCES = client.c
-client_LDADD = -L@prefix@/lib ../src/libxmlrpc.la
+client_LDADD = ../src/libxmlrpc.la
 
 server_SOURCES = server.c
-server_LDADD = -L@prefix@/lib ../src/libxmlrpc.la
+server_LDADD = ../src/libxmlrpc.la
 
 hello_client_SOURCES = hello_client.c
-hello_client_LDADD = -L@prefix@/lib ../src/libxmlrpc.la
+hello_client_LDADD = ../src/libxmlrpc.la
 
 hello_server_SOURCES = hello_server.c
-hello_server_LDADD = -L@prefix@/lib ../src/libxmlrpc.la
+hello_server_LDADD = ../src/libxmlrpc.la
 
 server_compliance_test_SOURCES = server_compliance_test.c
-server_compliance_test_LDADD = -L@prefix@/lib ../src/libxmlrpc.la
+server_compliance_test_LDADD = ../src/libxmlrpc.la
 
 memtest_SOURCES = memtest.c
-memtest_LDADD = -L@prefix@/lib ../src/libxmlrpc.la
+memtest_LDADD = ../src/libxmlrpc.la
 
Only in xmlrpc-epi-0.50.fixed/sample: client
Only in xmlrpc-epi-0.50.fixed/sample: hello_client
Only in xmlrpc-epi-0.50.fixed/sample: hello_server
Only in xmlrpc-epi-0.50.fixed/sample: memtest
Only in xmlrpc-epi-0.50.fixed/sample: sample
Only in xmlrpc-epi-0.50.fixed/sample: server
Only in xmlrpc-epi-0.50.fixed/sample: server_compliance_test
Only in xmlrpc-epi-0.50.fixed/src: Makefile
diff -ru --exclude missing --exclude ltmain.sh --exclude install-sh --exclude config.sub --exclude config.guess --exclude INSTALL --exclude COPYING --exclude mkinstalldirs --exclude Makfile --exclude Makefile.in --exclude aclocal.m4 --exclude configure --exclude '*.o' --exclude '*~' --exclude '*.lo' --exclude .libs --exclude .deps xmlrpc-epi-0.50/src/Makefile.am xmlrpc-epi-0.50.fixed/src/Makefile.am
--- xmlrpc-epi-0.50/src/Makefile.am	Sun Sep 16 22:27:27 2001
+++ xmlrpc-epi-0.50.fixed/src/Makefile.am	Thu Apr 11 16:27:17 2002
@@ -1,9 +1,7 @@
-INCLUDES=-I../liblm -I../expat/xmltok -I../expat/xmlparse -I/usr/local/ssl/include 
+INCLUDES=-I../liblm 
 
 lib_LTLIBRARIES = libxmlrpc.la
 
-libxmlrpc_la_LIBADD = ../expat/xmltok/libexpat_tok.la ../expat/xmlparse/libexpat_parse.la
-
 libxmlrpc_la_SOURCES = \
 	base64.c \
 	encodings.c \
Only in xmlrpc-epi-0.50.fixed/src: libxmlrpc.la
diff -ru --exclude missing --exclude ltmain.sh --exclude install-sh --exclude config.sub --exclude config.guess --exclude INSTALL --exclude COPYING --exclude mkinstalldirs --exclude Makfile --exclude Makefile.in --exclude aclocal.m4 --exclude configure --exclude '*.o' --exclude '*~' --exclude '*.lo' --exclude .libs --exclude .deps xmlrpc-epi-0.50/src/xml_element.c xmlrpc-epi-0.50.fixed/src/xml_element.c
--- xmlrpc-epi-0.50/src/xml_element.c	Mon May 14 20:31:28 2001
+++ xmlrpc-epi-0.50.fixed/src/xml_element.c	Thu Apr 11 16:39:19 2002
@@ -77,7 +77,15 @@
 
 #include "xml_element.h"
 #include "queue.h"
+#ifdef HAVE_XMLPARSE_H
 #include "xmlparse.h"
+#else
+#ifdef HAVE_EXPAT_H
+#include <expat.h>
+#else
+#error expat headers not found!!!
+#endif
+#endif
 #include "encodings.h"
 
 #define my_free(thing)  if(thing) {free(thing); thing = 0;}
@@ -668,7 +676,13 @@
          int col_num = XML_GetCurrentColumnNumber(parser);
          long byte_idx = XML_GetCurrentByteIndex(parser);
          int byte_total = XML_GetCurrentByteCount(parser);
+#ifdef HAVE_XMLPARSE_H
          const XML_LChar XMLPARSEAPI * error_str = XML_ErrorString(err_code);
+#else
+#ifdef HAVE_EXPAT_H
+         const XMLPARSEAPI(XML_LChar) * error_str = XML_ErrorString(err_code);
+#endif
+#endif
          if(byte_idx >= 0) {
              snprintf(buf, 
                       sizeof(buf),
diff -ru --exclude missing --exclude ltmain.sh --exclude install-sh --exclude config.sub --exclude config.guess --exclude INSTALL --exclude COPYING --exclude mkinstalldirs --exclude Makfile --exclude Makefile.in --exclude aclocal.m4 --exclude configure --exclude '*.o' --exclude '*~' --exclude '*.lo' --exclude .libs --exclude .deps xmlrpc-epi-0.50/src/xmlrpc.c xmlrpc-epi-0.50.fixed/src/xmlrpc.c
--- xmlrpc-epi-0.50/src/xmlrpc.c	Sun Sep 23 01:34:32 2001
+++ xmlrpc-epi-0.50.fixed/src/xmlrpc.c	Thu Apr 11 16:42:19 2002
@@ -105,7 +105,15 @@
 
 #include "queue.h"
 #include "xmlrpc.h"
+#ifdef HAVE_XMLPARSE_H
 #include "xmlparse.h"
+#else
+#ifdef HAVE_EXPAT_H
+#include <expat.h>
+#else
+#error expat headers not found!!!
+#endif
+#endif
 #include "base64.h"
 
 #include "xml_to_xmlrpc.h"
